segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DataFrameReader.java

```diff
@@ -45,6 +51,10 @@
     @Setter (AccessLevel.PROTECTED)
     private long lastReadSequenceNumber;
     private boolean closed;
+    private final String traceObjectId;
+    @Getter (AccessLevel.PACKAGE)
```

Comment:
are we using the getter or intended to use that?

Reply:
Good catch, you are right this is not necessary anymore.